src/bitcoin-cli.cpp

```diff
@@ -85,7 +85,7 @@ static bool AppInitRPC(int argc, char* argv[])
         }
 
         fprintf(stdout, "%s", strUsage.c_str());
-        return false;
+        if(argc < 2) return false; else exit(0);
```

Comment:
@laanwj I'm not sure if refactoring all nearby code would worth it. Wouldn't it be better to just `exit(0);` here then i.e. to show help by default when no command was provided without any error at all?